.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Auth0 Community
+    url: https://community.auth0.com/c/sdks/auth0-spa-js/71
```

Comment:
wrong sdk

Reply:
This has been updated in the internal docs too. This link used to exist but has since been removed.